models/scenario_testrun.go

```diff
@@ -0,0 +1,46 @@
+package models
+
+import "time"
+
+type TestrunStatus int
+
+const (
+	Up TestrunStatus = iota
+	Down
+	Unknown
+)
+
+func (t TestrunStatus) String() string {
+	switch t {
+	case Up:
+		return "Up"
```

Comment:
I'm fine with "up" and "down" as statuses, but let's please keep them lower case (we have a convention, in practice even if not in documentation, that statuses in DB/code are represented in lower snake case)